src/test/java/edu/cornell/kfs/tax/fixture/TaxOutputField.java

```diff
@@ -0,0 +1,46 @@
+package edu.cornell.kfs.tax.fixture;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import org.apache.commons.lang3.StringUtils;
+
+import edu.cornell.kfs.sys.CUKFSConstants;
+import edu.cornell.kfs.tax.batch.CUTaxBatchConstants.TaxOutputFieldType;
+import edu.cornell.kfs.tax.batch.xml.TaxOutputFieldV2;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.FIELD)
+public @interface TaxOutputField {
```

Comment:
Do you think it would be appropriate to rename this class to TaxOutputFieldFixture?

Reply:
Will rename accordingly.